.github/workflows/coverage-comment.yml

```
@@ -0,0 +1,239 @@
+        MIX_ENV: test
+        DATABASE_URL: postgres://postgres:postgres@localhost:5432/eve_tracker_test
+
+    - name: Run tests and generate coverage
+      run: mix coveralls.json
+      env:
+        MIX_ENV: test
+        DATABASE_URL: postgres://postgres:postgres@localhost:5432/eve_tracker_test
+
+    - name: Extract coverage data
+      id: coverage
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:7:3: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)